working-groups/green-reviews/charter.md

```diff
@@ -0,0 +1,53 @@
+# Green Reviews Working Group Charter
+
+## Introduction
+
+This document outlines the charter for the Green Reviews Working Group, a [CNCF Working Group](https://github.com/cncf/toc/blob/main/workinggroups/README.md) under the [CNCF TAG Environmental Sustainability](https://github.com/cncf/tag-env-sustainability).
+
+## Motivation
+
+CNCF projects are becoming increasingly concerned and aware of the challenges related to environmental sustainability. Their objective is to integrate an environmental sustainability project review in each release cycle. Furthermore, they strive to empower end users to assess the sustainability of the cloud native projects within their systems. 
+
+The purpose of the Green Reviews Working Group is to help CNCF projects to review their cloud native sustainability footprint.
+
+To support CNCF projects, project maintainers and contributors, there is a need for a systematic approach to assess every project's sustainability footprint. Implementing a structured review process for this purpose would greatly benefit the CNCF ecosystem, establishing it as a best practice. Such a process would enable stakeholders to comprehensively evaluate and enhance the sustainability of CNCF projects, promoting their long-term viability and environmental responsibility. 
+
+Green Reviews Working Group empowers the CNCF Environmental Sustainability TAG to pursue its goals of promoting environmentally sustainable practices in the CNCF ecosystem.
+
+## Scope
+
+- Act within the CNCF Environmental Sustainability TAG Charter. 
+- Provide metrics, guidelines, guidance, and processes for measuring and improving the sustainability footprint of CNCF projects.
+- Initiate a CNCF project review at the discretion of the Working Group.
+- Utilize existing tooling to conduct CNCF project sustainability footprint reviews.
+- Create minimal tooling to conduct CNCF project sustainability footprint reviews, for example, to automate certain processes, if existing tooling does not meet requirements, and if the CNCF Environmental Sustainability TAG formally approved this.
+- Be a point of contact for project maintainers to understand the outcome of the projects’ reviews.
+
+## Goals
+
+- Provide resources through a review process and methodology to help CNCF projects to evaluate their sustainability footprint[^1] and integrate environmental sustainability practices. 
+- Collaborate with CNCF project maintainers to produce meaningful sustainability footprint reviews.
+- Communicate the results of the projects’ cloud native sustainability reviews.
+
+
+## Non-Goals
+
+- Build tooling for measuring or monitoring sustainability-related metrics from scratch.
+- Contribute code to existing tooling and cloud native sustainability projects such as bug fixes or features.
```

Comment:
For my understanding -- can you clarify the amount of tooling (maybe through some examples?) that will be in scope for the WG? I'm a little confused because:

The `Scope` section calls out:

> Create minimal tooling to conduct CNCF project sustainability footprint reviews, for example, to automate certain processes, if existing tooling does not meet requirements, and if the CNCF Environmental Sustainability TAG formally approved this.

and `Deliverables` mention:

> Develop testing infrastructure to utilise existing tooling to measure the sustainability footprint of CNCF projects.

Does this involve writing tooling to build the test infra?

> Provide visualisation to communicate the outcomes of the sustainability footprint CNCF project reviews, for example, through dashboards for carbon and energy monitoring.

Will existing tooling be used to build dashboards or does this involve building additional tooling?

Reply:
Right, good call out, we talked about tooling for a bit during the two [WG discussion meetings](https://www.youtube.com/watch?v=bAZEWr85Ggg&list=PL_3_oSSuNpD4DDVxVF4zx8HklbOUP9HFP). For now, the focus lies on the discovery part, which is essentially about understanding how we want to review the sustainability footprint of CNCF projects. As we don't know the step-by-step process yet, we are unsure about the automation options we have down the road. It's highly likely that we will benefit from automation, that's why we mention it in the first place in the charter. We expressed worries during our calls that this may lead to working on an entire test suite or something similar, which would lead off from the core course of this WG. So we open up the option of developing automation tooling, but we limit it by phrasing it "minimal automation tooling". With minimal automation tooling, we follow the idea of adopting existing libraries and projects if possible and only, for example, add a script or small-sized programs on top of. Maybe we can clarify this, what do you think?

Reply:
> Maybe we can clarify this, what do you think?

Sounds good!

Reply:
cc @nikimanoledaki @guidemetothemoon

Reply:
@nikhita @leonardpahlke @nikimanoledaki thanks for providing detailed reflections, I totally agree that this point needed additional clarification. Based on this discussion I've attempted to improve clarification of the respective points both in the Scope and Deliverables section. Please take a look when you have time and let me know if anything else should be changed or improved😊

Reply:
@guidemetothemoon thanks! LGTM :shipit: